src/app/components/ThemeProvider/fontFamilies.ts

```diff
@@ -26,5 +26,5 @@ export const PADAUK = 'Padauk, Arial, Verdana, Geneva, Helvetica, sans-serif';
 export const REITH_SANS = 'ReithSans, Helvetica, Arial, sans-serif';
 export const REITH_SERIF = 'ReithSerif, Helvetica, Arial, sans-serif;';
 export const REITH_QALAM =
-  '"BBC Reith Qalam", Arial, Verdana, Geneva, Helvetica, sans-serif';
+  '"Times New Roman", "BBC Reith Qalam", Arial, Verdana, Geneva, Helvetica, sans-serif';
```

Comment:
this needs to go the other way around

We want Qalam to be preferred, but while user is downloading the font the 2nd font will be picked up